libraries/lib-datahandler/src/test/java/org/eclipse/sw360/datahandler/couchdb/AttachmentContentDownloaderTest.java

```diff
@@ -88,7 +89,9 @@ public void testABlackHoleUrl() throws Exception {
                 } catch (ExecutionException e) {
                     Throwable futureException = e.getCause();
                     assertThat(futureException, is(notNullValue()));
-                    assertTrue(futureException instanceof SocketException || futureException instanceof SocketTimeoutException);
+                    assertTrue(futureException instanceof SocketException ||
```

Comment:
Just IOException?
SocketException and SocketTimeoutException both extends IOException

Reply:
good point!